student_div.c

```diff
@@ -1,19 +1,20 @@
 #include<stdio.h>
-Int main()
+int main()
 {
-    Int m1,m2,m3,m4,m5,per;
-    Printf("enter marks in five subjects");
-    Scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
+    int m1,m2,m3,m4,m5;
+    float per;
+    printf("enter marks in five subjects: ");
+    scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
 
-     Per=(m1+m2+m3+m4+m5)/500*100;
+    per=(m1+m2+m3+m4+m5)*100/500; 
+    printf("Percentage: %f\n", per);
+    if(per>=60)
+          printf("first division\n");
      
-    If(per>=60)
-          Printf("first division\n");
-     
-    If((per>=50)&&(per<50))
-          Printf("second division\n");
+    else if((per>=50)&&(per<60))
```

Comment:
Line 14 would work the same way even when you write something like 
else if (per >= 50)

Reply:
Yes sir!! I was just trying to make the corrections and as this step also had one error (50 was written instead of 60), I didn't change the format for a better understanding of Komal. Should I change it?

Reply:
You should change it coz that it way it would be a little more compute efficent.

Reply:
Okay sir!